src/components/projectsettings.test.ts

```diff
@@ -92,3 +96,38 @@ test('update project name', async () => {
     path: '/org/org01/projects/newproj01.proj/settings',
   });
 });
+
+test('update org project membersCanPerformRunActions', async () => {
+  await flushPromises();
+
+  const membersCanPerformRunActions = wrapper.find<HTMLInputElement>(
+    '[data-test="membersCanPerformRunActions"]'
+  );
+  const updateProjectButton = wrapper.find<HTMLInputElement>(
+    '[data-test="updateProjectButton"]'
+  );
+
+  membersCanPerformRunActions.setValue(true);
+  updateProjectButton.element.click();
+
+  await flushPromises();
```

Comment:
here you aren't testing the real update. But that the element is checked and this happens when you do the setValue(true)

Reply:
I've tried to implement the test in different way. Thank you.

Reply:
Sorry but I don't understand this tests. You shouldn't rely on the html element state since you don't know who set their value.

So, if you're testing `update org project membersCanPerformRunActions` then you should just check that the update api has been called like in other tests with the right values (so check that the api request has the required values).